doc/manuals/orion-api.md

```diff
@@ -405,6 +438,14 @@ The rules are:
 In addition to the above rules, given NGSIv2 server implementations could add additional syntactical
 restrictions in those or other fields, e.g., to avoid cross script injection attacks.
 
```

Comment:
Discussed at skype, two things in this part:

* Avoid "neutral references" (i.e. NGSI2 sever implementations) as this document fully refers to Orion
* Move forbidden_characters.md#general-restrictions into this document. In the same operation, forbidden_characters.md#specific-restrictions-for-id-fields can be removed, as it is doesn't have any sense